notes/2020-10-02.md

```diff
@@ -30,52 +30,58 @@ ca_dds_df
 ```{code-cell} ipython3
 print(ca_dds_df)
 ```
-
+we can do df.index to look at what the index is in our dataframe
 ```{code-cell} ipython3
 ca_dds_df.index
 ```
-
+columns are tuple values. We can index to the first column as below. As we can see by running this code, this column has three names initially. 
 ```{code-cell} ipython3
 ca_dds_df.columns[0]
 ```
-
+By the following code we can assign the first column as the index of original dataframe. If we assign "False" to the "inplace", it will make the changes in a new dataframe.
 ```{code-cell} ipython3
 ca_dds_df.set_index(ca_dds_df.columns[0],inplace=True)
 ```
-
+The updated dataframe looks like below. We still have the name of index inappropriate and aslo there are empty rows. 
 ```{code-cell} ipython3
 ca_dds_df
 ```
-
+we can drop rows with "all" or "any",  "NaN" values. We would choose "all", since if we choose "any", we will loose many rows here and we do not want it.
```

Comment:
```suggestion
we can drop rows with rows or columns with `NaN` values in two ways: "all" or "any". The default is `any` which will drop a row if it has _any_ values that are `NaN`. We will use "all" instead, because we only want to drop the rows with no data in them.
```

Your description here is close, but not detailed.